### DISCUSSION TITLE
Staggering Group Reinforce Strategy

### DISCUSSION THREAD

**Discussion by zishanali06** [coccoinomane/crabada.py#105] [Q&A] [answered]
Hey @yigitest or @coccoinomane, so if I am going to use the staggered team grouping. How do I specify the reinforce strategy for that group? Like USER_1_TEAM_1_REINFORCE_STRATEGY should be changed to something else correct? Maybe USER_1_STAGGER_GROUP_1_REINFORCE_STRATEGY or is it something else?

Also thanks a bunch for such a fast swimmer update!

**Comment by yigitest** [comment, marked as answer]
Hi, 

1. Group your teams:  `USER_1_STAGGER_GROUP_1_TEAMS=12345,5678,123,7654`
2. set a reinforce strategy:  `USER_1_TEAM_1_REINFORCE_STRATEGY=HighestMpFromInventory`  
  _You'll probably want to use a fallback strategy as well: `USER_1_TEAM_1_REINFORCE_STRATEGY=HighestMpFromInventory, HighestMpHighCost`_
3. Repeat step [2] for all of your teams in the group.

**Reply by zishanali06** [reply]
Thanks @yigitest!

Ok so is this below how I need to have it in the env file?

```
USER_1_STAGGER_GROUP_1_TEAMS=2001,2002,2003   (do I need to have the “” around these numbers? So have it like “2001,2002,2003”?)
USER_1_STAGGER_DELAY=35  (same here do I need to have the “”?)
USER_1_TEAM_1=“2001”
USER_1_TEAM_1_REINFORCE_STRATEGY="HighestMpFromInventory, HighestMpHighCost"
USER_1_TEAM_1_TASK=“mine”.  (Do I still need this line?)
USER_1_TEAM_1_REINFORCEMENT_TO_PICK="8" 
USER_1_TEAM_2=“2002”
USER_1_TEAM_2_REINFORCE_STRATEGY="HighestMpFromInventory, HighestMpHighCost"
USER_1_TEAM_2_TASK=“mine”
USER_1_TEAM_2_REINFORCEMENT_TO_PICK=“9” 
USER_1_TEAM_3=“2003”
USER_1_TEAM_3_REINFORCE_STRATEGY="HighestMpFromInventory, HighestMpHighCost"
USER_1_TEAM_3_TASK=“mine”
USER_1_TEAM_3_REINFORCEMENT_TO_PICK=“10” 
```

**Reply by yigitest** [reply]
I don't use quotation marks at all, it could be platform dependent i don't know. 


> USER_1_TEAM_1_TASK=“mine”.  (Do I still need this line?)  

Yep

**Reply by coccoinomane** [reply]
Hmm, these do not look like standard quotation mark:

<img width="259" alt="Markup 2022-05-15 at 20 01 57" src="https://user-images.githubusercontent.com/3920436/168487239-3f4ca96a-c2c6-4730-b1ee-34a20f896fb8.png">

A standard quotation mark should look like this `"`.

I'd suggest to remove the quotation marks altogether, in order to avoid issues 🙂

Cheers,
Cocco
